packages/components/psammead-story-promo/src/index.stories.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { Fragment } from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import { inputProvider } from '@bbc/psammead-storybook-helpers';
+import Img from '@bbc/psammead-image';
+import { latin } from '@bbc/gel-foundations/scripts';
+import Timestamp from '@bbc/psammead-timestamp';
+import notes from '../README.md';
+import StoryPromo, { Headline, Summary } from './index';
+
+const ImageComponent = (
+  <Img
+    alt={text('Image alt text', '2019-03-01T14:00+00:00')}
+    src={text(
+      'Image src',
+      'https://ichef.bbci.co.uk/news/660/cpsprodpb/909B/production/_106491073_gettyimages-909260196.jpg',
+    )}
+    width="640"
+  />
+);
+
+// eslint-disable-next-line react/prop-types
+const TextComponent = ({ headlineText, summaryText }) => (
```

Comment:
Not convinced on the naming `TextComponent`

Reply:
Text is containing a headline, summary and timestamp. Could this be `PromoInfo` or similar

Reply:
renamed :)

Reply:
Do we need to pass the image `width`? Are we really using that value?

Reply:
@DenisBBC Annoyingly its a required prop https://github.com/bbc/psammead/blob/latest/packages/components/psammead-image/README.md at the moment.